inner-scripts/kubestellar

```diff
@@ -159,14 +217,20 @@ function create_or_replace() { # usage: filename
     filename="$1"
     kind=$(grep kind: "$filename" | head -1 | awk '{ print $2 }')
     name=$(grep name: "$filename" | head -1 | awk '{ print $2 }')
-    if kubectl get "$kind" "$name" &> /dev/null
+    if kubectl get "$kind" "$name" # &> /dev/null
     then eval kubectl replace -f "$filename" $verbdir
     else eval kubectl create -f "$filename" $verbdir
     fi
 }
 
-# current ws at start does not matter, is root:compute on return
 function ensure_root_compute_configd() {
+    # TODO: remove kcp dependency in PR3 once we switch to kube-bind.# TODO: remove kcp dependency in PR3 once we switch to kube-bind.
```

Comment:
There is some kind of editing botch here, I see the same comment repeated twice on this line.